### DISCUSSION TITLE
Zsh completion with fzf-tab doesn't work with dotnet completion command

### DISCUSSION THREAD

**Discussion by iabdelkareem** [Aloxaf/fzf-tab#290] [Q&A] [answered]
I enabled zsh completion and added fzf-tab via zinit. It works as expected. I added some custom/extra completions e.g., aws cli and it also worked as expected.

However, when I tried to enable the completion for dotnet as recommended it didn't work with fzf. Please see the screenshots below and the code from my .zshrc for the completions.

What am I missing to get dotnet cli work with fzf-tab completion menu?

[![autoload bashcompinit; bashcompinit
autoload -Uz compinit; compinit

```
_comp_options+=(globdots) # With hidden files
_dotnet_zsh_complete()
{

local completions=("$(dotnet complete "$words")")
reply=( "${(ps:\n:)completions}" )

}

compctl -K _dotnet_zsh_complete dotnet
complete -C "/usr/local/bin/aws_completer" aws
```

This is how it looks with system commands e.g., cp when I press TAB
![https://i.stack.imgur.com/EQS2W.png](https://i.stack.imgur.com/EQS2W.png)

This is how it looks with AWS CLI when I press TAB AWS completion
![2](https://i.stack.imgur.com/nFwQ7.png)

This is how it looks with dotnet CLI when I press TAB enter image description here
![3](https://i.stack.imgur.com/H8Urm.png)

**Comment by Aloxaf** [comment, marked as answer]
fzf-tab only supports compsys, compctl is old fashion and is not supportted.

Here is a brief documentation to compsys: https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org

**Reply by iabdelkareem** [reply]
Thank you ^^
